src/libraries/System.Formats.Cbor/Directory.Build.props

```diff
@@ -0,0 +1,7 @@
+﻿<Project>
+  <Import Project="..\Directory.Build.props" />
+  <PropertyGroup>
+    <StrongNameKeyId>Microsoft</StrongNameKeyId>
+    <IsNETCoreApp>false</IsNETCoreApp>
```

Comment:
I feel like we usually just leave the property out all together rather than specify it as false.

Reply:
Sure, I felt it's one of this worth making explicit.